torch/fx/examples/custom_tracer.py

```diff
@@ -0,0 +1,115 @@
+import torch
+from torch.fx import symbolic_trace, Tracer, Graph, GraphModule, Node
+from typing import Any, Callable, Dict, Optional, Tuple, Union
+
+"""
+How to Create and Use Custom Tracers
+
+`Tracer`--the class that implements the symbolic tracing functionality
+of `torch.fx.symbolic_trace`--can be subclassed to override various
+behaviors of the tracing process. In this tutorial, we'll demonstrate
+how to customize the symbolic tracing process using some handwritten
+Tracers. Each example will show that, by simply overriding a few methods
+in the `Tracer` class, you can completely alter the Graph produced by
```

Comment:
Not sure if completely altering it is a selling point, I would just say "you can alter the Graph"

"Completely alter" would necessarily bring up questions of "so is tracing not generally usable downstream?"